server/controllers/clubProfileController.js

```diff
@@ -82,23 +84,37 @@ module.exports = {
     },
     create: function(req, res) {
         // TODO; req.body contains profile information
-        clubProfile.create(req.body)
-                .then(newclubProfile => res.json(newclubProfile))
+        clubProfile.create(req.body) 
+                .then(newclubProfile => {
+                  //set new profile's acc ID to the given accID
+                  newclubProfile.clubAccountId = req.params.accountId;
+                  //get the account assosiated with given accID and set its profile Id to the new club profile Id
+                  var newClubAccount = getById(req.params.accountId,res);
+                  newClubAccount.clubProfileId = newclubProfile.id;
```

Comment:
as a matter of fact `newClubAccount` and `newclubProfile` are both mongoose query objects instead of real js objects, so to access a field of the *result* of the query, you might need to parse it into JSON object first (i.e. `JSON.parse(JSON.stringify(newclubProfile))`. 

On the other hand, to set the field of a query, you can actually combine this with the last line of code and just use a `findByIdAndUpdate`. you can find more about this on the mongoose docs!